.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
One more thing ... I pushed a change to package.json that adds a `coverage` script. Merge that with your branch and try running `npm run coverage`. Then open `coverage/lcov-report/index.html` in a browser. It clearly shows what isn't yet covered by tests. Increasing the coverage will be good practice for the Monsanto projects.